content-filter-pattern-kinesis-cdk/src/bin/src.ts

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { ContentFilterStack } from '../lib/content-filter-stack';
```

Comment:
Think we missing the lib directory @dominik-richter

Reply:
Thanks for pointing it out. Changed the gitignore file to allow .js files in lib/lambda